channels/rdpgfx/server/rdpgfx_main.c

```diff
@@ -1104,9 +1104,9 @@ static UINT rdpgfx_recv_cache_import_offer_pdu(RdpgfxServerContext* context, wSt
 	/* cacheEntriesCount (2 bytes) */
 	Stream_Read_UINT16(s, pdu.cacheEntriesCount);
 
-	if (pdu.cacheEntriesCount <= 0)
+	/* 2.2.2.16 RDPGFX_CACHE_IMPORT_OFFER_PDU */
+	if (pdu.cacheEntriesCount > 5462)
```

Comment:
2.2.2.16 says "MUST be less than 5462" so this check should be `>=`

Reply:
good catch